models/user.rb

```diff
@@ -10,9 +10,12 @@ class User < Base
 
   RESET_WINDOW = 60 * 15 # 15 minutes
 
-  SETTINGS = %w[
-    notifications bg font bg2 font2 red_logo white yellow green brown gray red blue
-  ].freeze
+  SETTINGS = 4.times.flat_map do |n|
+    %w[color dash width].map do |prop|
+      "r#{n}_#{prop}"
```

Comment:
i don't want "r#{n}_#{props}" in the code if possible

Reply:
Well, the settings do need a name. The other option would be hashes, but as I already wrote in my PR comment: 

> no hash with route settings because of problems with settings-array in models/user.rb and accessing values with color_for/settings_for. I wouldn’t mind rewriting these parts, but would likely need a lot of help.